src/rules/no-reversed-media-syntax.js

```diff
@@ -143,7 +143,9 @@ export default {
 		}
 
 		return {
-			"heading html,inlineCode"(node) {
+			":matches(heading, paragraph, tableCell) :matches(html, inlineCode)"(
```

Comment:
```suggestion
			"heading html,heading inlineCode"(
```

I think simply adding `heading` here without modifying the existing logic would make our intention clearer.

This applies to the `paragraph` and `tableCell` ESQuery selectors.